ext/fetch/lib.rs

```diff
@@ -467,9 +465,8 @@ impl HttpClientResource {
 #[serde(rename_all = "camelCase")]
 #[serde(default)]
 pub struct CreateHttpClientOptions {
-  ca_stores: Option<Vec<String>>,
-  ca_file: Option<String>,
-  ca_data: Option<ByteString>,
+  #[serde(default)]
```

Comment:
I admit I don't understand what this changes. It's also specified on the type.

Reply:
It should force `caCerts: undefined` to be deserialized as `caCerts: []`. Essentially use `Default::default` when the field is falsy or not present.